src/components/map/test/unit/vl-map-overview-map.test.html

```diff
@@ -1,48 +1,46 @@
 <!DOCTYPE html>
 <html>
 
-<head>
+  <head>
     <meta charset="utf-8">
     <script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
     <script src="/node_modules/web-component-tester/browser.js"></script>
 
     <script type="module" src="../../../../../lib/index.js"></script>
 
-  <body>
-    <test-fixture id="vl-map-overview-map-fixture">
-      <template>
-        <vl-map>
-          <vl-map-overview-map></vl-map-overview-map>
-          <vl-map-baselayer
-            data-vl-url="https://localhost"
-            data-vl-layer="layername_1"
-            data-vl-title="layer title 1"
-          ></vl-map-baselayer>
-        </vl-map>
-      </template>
-    </test-fixture>
+    <body>
+      <test-fixture id="vl-map-overview-map-fixture">
+        <template>
+          <vl-map>
+            <vl-map-overview-map></vl-map-overview-map>
+            <vl-map-baselayer data-vl-url="https://tile.informatievlaanderen.be/ws/raadpleegdiensten/wmt"
+              data-vl-layer="grb_bsk"
+              data-vl-title="GRB basis laag"></vl-map-baselayer>
+          </vl-map>
+        </template>
+      </test-fixture>
 
-    <script>
-      suite("vl-map-overview-map", () => {
-        setup((done) => {
-          customElements.whenDefined("vl-map-overview-map").then(() => done());
-        });
+      <script>
+suite("vl-map-overview-map", () => {
+  setup((done) => {
+    customElements.whenDefined("vl-map-overview-map").then(() => done());
+  });
 
-        test("de overview map wordt goed geconfigureerd", (done) => {
-          const element = fixture("vl-map-overview-map-fixture");
-          const vlMapOverviewMapElement = element.querySelector(
-            "vl-map-overview-map"
-          );
-          (async () => {
-            const bevatOverviewMap = (map) =>
-              map.getControls().getArray().includes(map.overviewMapControl);
-            while (!bevatOverviewMap(vlMapOverviewMapElement._map)) {
-              await new Promise((resolve) => setTimeout(resolve, 100));
-            }
-            done();
-          })();
-        });
-      });
-    </script>
-  </body>
+  test("de overview map wordt goed geconfigureerd", (done) => {
+    const element = fixture("vl-map-overview-map-fixture");
+    const vlMapOverviewMapElement = element.querySelector(
+      "vl-map-overview-map"
+    );
+    (async () => {
+      const bevatOverviewMap = (map) =>
+      map.getControls().getArray().includes(map.overviewMapControl);
+      while (!bevatOverviewMap(vlMapOverviewMapElement._map)) {
+        await new Promise((resolve) => setTimeout(resolve, 100));
+      }
```

Comment:
Hebben we (ondertussen) geen awaitUntil ergens in de vl-core zitten waarmee dit "properder" kan?

Reply:
Aangepast.